sentry/src/test/java/io/sentry/config/ClasspathPropertiesLoaderTest.kt

```diff
@@ -0,0 +1,58 @@
+package io.sentry.config
+
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.whenever
+import io.sentry.NoOpLogger
+import java.io.IOException
+import java.nio.charset.Charset
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+class ClasspathPropertiesLoaderTest {
+    private class Fixture(fileName: String = "sentry.properties", content: String? = null, throws: Boolean = false) {
```

Comment:
why don't we use the `getSut` method to pass the params and init the proper class like all the other tests?

Reply:
👍